### PR TITLE
Parsed issue

### PR DESCRIPTION
**Pull request by Danwhy**
Adds function to separate out the relevant parts of the issue object that we want, and updates README to show data structure.

Function for adding user to database and fetching issues is in the process of being tested.

### CODE REVIEW

File: test/gh_to_db_parser.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+var test             = require("tape");
+var parseIssues      = require("../api/parse_issues.js");
+var issueArray       = require("../example_issues.json");
+
+var parsedIssue1 = {
```
Comment (NataliaLKB) [inline test/gh_to_db_parser.test.js]: Please consider creating a `fixtures` directory to put in data like this and then require it. 

As sample data is something that often is reused within other tests best to keep it in one place for easy access.

Also, consider the readability. When you open a test file you expect to see tests and it is a bit confusing to see data first thing.

**Comment by nelsonic** [comment]
@Danwhy please make the change recommended by @NataliaLKB https://github.com/dwyl/tudo/pull/68#discussion-diff-35345438 and push the change to this branch.
otherwise, looks good. thanks! :+1:

**Comment by Danwhy** [comment]
@nelsonic @NataliaLKB Done! Thanks

**Comment by nelsonic** [comment]
Thanks @Danwhy 
Please check why Travis is unhappy: https://travis-ci.org/dwyl/tudo/builds/72331780

**Comment by nelsonic** [comment]
@Danwhy thanks for investigating. we will need to _understand_ why we aren't getting 100% Coverage on Travis-CI. but given that all tests pass and we have 100% on all local machines, we are merging this. thanks!

**Pull request merged**